### PR TITLE
ZENKO-3410: fix eslint warnings dependencies

### PR DESCRIPTION
**Pull request by romainlancia**
**Problem:**
Screenshot from 2021-04-23 11-55-30

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: src/react/account/AccountList.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ function AccountList({ accountList, accountIndex }: Props) {
         if (listRef && listRef.current && accountIndex > -1) {
             listRef.current.scrollToItem(accountIndex, 'smart');
         }
-    }, [listRef]);
+    }, [listRef, accountIndex]);
```
Comment (nicolas2bert) [inline src/react/account/AccountList.jsx]: `accountIndex` was not mentioned on purpose. 
We want to scroll to the selected item only once on mount. 
We can add a comment. :)

**Comment by alexis-ld** [comment]
@nicolas2bert sorry for the delay, we missed the comment :) 

I double checked this issue was still up to date and it is. We took care of the required changes and it should be ready to merge :)

**Review by miniscruff — APPROVED** [review]

**Comment by elenamarcova** [comment]
@nicolas2bert could you please review this PR ?

**Comment by elenamarcova** [comment]
@naren-rajendran could you please review this PR if @nicolas2bert is not available ?

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**